core/src/main/scala/io/finch/request/package.scala

```diff
@@ -108,11 +104,10 @@ package object request {
     req.cookies.get(cookie)
 
   private[request] def requestUpload(upload: String)(req: Request): Option[FileUpload] = {
-    import com.twitter.finagle.httpx.netty.Bijections._
-    val decoder = new HttpPostRequestDecoder(from(req))
```

Comment:
Can you also fix `requestParam` to use `Multipart` instead of Netty's `HttpPostRequestDecoder`?